### ISSUE TITLE
Save contrast mode in localStorage for web

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#3187]
### Describe the Problem
Contrast mode should be saved in local storage to be able to use the theme consistently during the interaction with the website

### Describe Your Preferred Solution

Add contrast mode to localStorage

**Comment by steffenkleinle** [comment]
Fixed in #3212
